pkg/processor/runtime/shell/runtime.go

```diff
@@ -146,6 +154,17 @@ func (s *shell) getCommandArguments(event nuclio.Event) string {
 	return ""
 }
 
+func (s *shell) getResponseHeadersFromConfiguration() map[string]interface{} {
+	if responseHeaders, responseHeadersExists := s.configuration.Spec.RuntimeAttributes["responseHeaders"]; responseHeadersExists {
+		s.Logger.DebugWith("Found headers in function spec that will be added to all responses",
+			"headers", responseHeaders)
+		return responseHeaders.(map[string]interface{})
```

Comment:
I think we have to check if this type assertion in itself is okay - what if I give you `runtimeAttributes->responseHeaders: "blaablabl"`?